chpl/chpl-api/e2e/collections/certified-product-controller.postman_collection.json

```diff
@@ -1,54 +1,922 @@
+					},
+					"response": []
+				},
+				{
+					"name": "GET /{chplPrefix}-{identifier} - valid API Key returns RWT fields",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									"const chplPrefix= pm.iterationData.get(\"chplPrefix\");\r",
+									"const identifier= pm.iterationData.get(\"identifier\");\r",
+									"pm.environment.set(\"chplPrefix\",\"chplPrefix\");\r",
+									"pm.environment.set(\"identifier\",\"identifier\");"
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"/{chplPrefix}-{identifier} end point response should have all RWT fields\", function () {\r",
+									"    var resultexp = pm.response.json()\r",
+									"    pm.expect(resultexp).to.have.property('rwtPlansUrl');\r",
+									"    pm.expect(resultexp).to.have.property('rwtPlansCheckDate');\r",
+									"    pm.expect(resultexp).to.have.property('rwtResultsUrl');\r",
+									"    pm.expect(resultexp).to.have.property('rwtResultsCheckDate');\r",
+									"    pm.expect(resultexp).to.have.property('rwtEligibilityYear');\r",
+									"});\r",
+									"pm.test(\"All RWT fields should be null for 2011 and 2014 listings\", function() {\r",
+									"    var jsonData = pm.response.json();\r",
+									"    pm.expect(jsonData.rwtPlansUrl).eq(null);\r",
+									"    pm.expect(jsonData.rwtPlansCheckDate).eq(null);\r",
+									"    pm.expect(jsonData.rwtResultsUrl).eq(null);\r",
+									"    pm.expect(jsonData.rwtResultsCheckDate).eq(null);\r",
+									"    pm.expect(jsonData.rwtEligibilityYear).eq(null);\r",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "API-Key",
+								"value": "{{apiKey}}",
+								"type": "text"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "{{url}}/rest/certified_products/{{chplPrefix}}-{{identifier}}",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"rest",
+								"certified_products",
+								"{{chplPrefix}}-{{identifier}}"
+							]
+						},
+						"description": "GET /{chplPrefix}-{identifier} end point response returns status 200 and have all RWT fields"
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "GET : Certified_products: non-data driven tests",
+			"item": [
+				{
+					"name": "GET : /{certifiedProductId}",
+					"item": [
+						{
+							"name": "GET /{certifiedProductId} - valid API Key return all RWT fields",
+							"event": [
+								{
+									"listen": "prerequest",
+									"script": {
+										"exec": [
+											"const certifiedProductId= pm.iterationData.get(\"certifiedProductId\");\r",
```

Comment:
If this is in the non-data driven tests, why does it look like it's getting a certifiedProductId from somewhere in the this exec block?

Reply:
Fixed and added data driven tests for /certifiedProductId end point.